lib/HTFeed/Stage/Download.pm

```diff
@@ -63,7 +63,7 @@ sub download{
 
     if( $response->is_success() ){
         my $size = (-s $pathname);
-	$self->{job_metrics}->add("ingest_download_bytes", $size);
+	$self->{job_metrics}->add("ingest_download_bytes_r_total", $size);
```

Comment:
I forget why the `r` / `w` thing came up; I don't think it's necessary here insofar as download implies a certain thing (both reading bytes over the network and writing them out to disk); doesn't need to block merge.

Reply:
I thought you wanted me to specify _r/_w in general, and then it made sense to stick with _r even when implied, for consistency.

Reply:
Right, I think just for download it's kind of dual purpose (reading from the network, writing to disk), but it's not a big deal.